src/lib/components/LeftSidebarNav.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+    import Link from './Link.svelte';
+    import PlayIcon from '$lib/elements/icons/Play.svelte';
+
+    export let npub: string;
+</script>
+
+<div class="flex flex-col gap-4 mt-8 font-semibold text-lg">
+    <Link
+        href={`/${npub}/listen`}
+        klass="text-white whitespace-nowrap"
+        activeKlass="text-violet-500"
+    >
+        <PlayIcon />
+
+        <span class="whitespace-nowrap">Listen</span>
+    </Link>
+
+    <Link
+        href={`/${npub}/playlists`}
+        klass="text-white whitespace-nowrap"
+        activeKlass="text-violet-500"
+    >
+        <svg
```

Comment:
Let's move this to an `icon.svelte` file? there's an `icons` directory iirc

Reply:
Good catch. I didn't seem them in the copy/paste but they definitely belong in the Icon dir.

Reply:
👍 cool, yeah it was a quick hack dropping them there before I added the `icons` dir